docs/en/observability/observability-ai-assistant.asciidoc

```diff
@@ -193,6 +193,13 @@ This opens the AI Assistant flyout, where you can ask the assistant questions ab
 [role="screenshot"]
 image::images/obs-ai-chat.png[Observability AI assistant chat, 60%]
 
+[IMPORTANT]
+====
+Asking questions about your data requires `function calling`, which enables LLMs to reliably interact with third-party generative AI providers to perform searches or run advanced functions using customer data.
+
+When the {observability} AI Assistant performs searches in the cluster, the queries are run with the same level of permissions of the user.
```

Comment:
I think you meant to say "as the user"? Otherwise you probably want to say, "the queries are run with the permissions of the user." (I like the first version better.)
```suggestion
When the {observability} AI Assistant performs searches in the cluster, the queries are run with the same level of permissions as the user.
```